src/main/java/de/tum/p2p/api/netty/handle/OnionTunnelReadyMessageHandler.java

```diff
@@ -0,0 +1,25 @@
+package de.tum.p2p.api.netty.handle;
+
+import de.tum.p2p.api.netty.forwarding.OnionTunnelReadyMessage;
+import de.tum.p2p.onion.forwarding.OnionForwarder;
+import io.netty.channel.ChannelHandlerContext;
+import io.netty.channel.SimpleChannelInboundHandler;
+
+public class OnionTunnelReadyMessageHandler extends SimpleChannelInboundHandler<OnionTunnelReadyMessage> {
```

Comment:
I think this handler is useless.  OnionTunnelReadyMessage is an outbound message and OnionTunnelBuildMessageHandler supersedes this one (sends OnionTunnelReadyMessage on tunnel built)